frames/src/main/java/com/checkout/frames/base/usecase/UseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.checkout.frames.base.usecase
+
+/**
+ * Used for creation of use case
+ */
+internal interface UseCase<D, T> {
```

Comment:
UseCase<D, T> already exist in headless SDK we can reuse it.
Are there any obstacles to it?

Reply:
It's internal in headless SDK. As we discussed earlier I have created a base for frames.

Reply:
We can make it not internal, public f.e. with @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP).
As far as I remember we discussed implementation via usecase, but not creation of new base usecase interface.